hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamer.java

```diff
@@ -1377,34 +1399,39 @@ private void testAsyncClusteringService(HoodieRecordType recordType) throws Exce
     UtilitiesTestBase.Helpers.deleteFileFromDfs(fs, tableBasePath);
   }
 
+  @Timeout(600)
+  @Test
+  public void testAsyncClusteringServiceWithConflictsAvro() throws Exception {
```

Comment:
so whenever it's flaky it's only for AVRO record type?

Reply:
nope. does not matter. I did not find any real necessity to run this test for both spark and avro. and hence.